src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+interface Props {
+  movie: Movie;
+}
+
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+  } = movie;
+
+  return (
+    <div className="card" data-cy="Movie">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img data-cy="MovieImage" src={imgUrl} alt="Film logo" />
```

Comment:
use `title` for alt text
```suggestion
          <img data-cy="MovieImage" src={imgUrl} alt={title} />
```